modules/promotions/admin-menu-items/form-submissions-promotion-item.php

```diff
@@ -21,15 +19,15 @@ public function get_page_title() {
 		return esc_html__( 'Submissions', 'elementor' );
 	}
 
-	public function get_promotion_title() {
+	protected function get_promotion_title(): string {
```

Comment:
I see that the access modifiers were changed only in this file.
Maybe it is better to change all the access modifiers for the desired function in the rest of the desired classes?